api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -391,17 +396,36 @@ public Workspace updateWorkspaceAcls(
 
   @Override
   @Transactional
-  public Workspace saveAndCloneCohortsAndConceptSets(Workspace from, Workspace to) {
+  public Workspace saveAndCloneCohortsConceptSetsAndDataSets(Workspace from, Workspace to) {
```

Comment:
nit: it might be time to come up with a collective term for workspace "stuff", that is top level items like datasets, cohorts, and concept sets. Then the function would be `saveAndCloneWorkspaceStuff()` or whatever.

Reply:
I don't have a good word for that that is descriptive yet. Maybe `workspaceObjects`? I can think on it while doing the other edits, but if you have any ideas, happy to take them.